### ISSUE TITLE
Multiple databases support

### ISSUE THREAD

**Issue by yukukotani** [Quramy/prisma-fabbrica#246]
Thanks for building such a great library!

We have multiple databases on the same application and would like to initialize prisma-fabbrica multiple times with different PrismaClient to connect them.

But since fabbrica currently stores PrismaClient instance [as a singleton](https://github.com/Quramy/prisma-fabbrica/blob/f9da749a1ba35153ac3644dc73c12cd78880fc1f/packages/prisma-fabbrica/src/clientHolder.ts), every initialization overrides previous one.

Any chance to support multiple databases?

**Comment by Quramy** [comment]
Are there multiple `schema.prisma` files in your project ?

I can add an option to separate `prisma` instance for each generated fabbrica file. 

```ts
import { initialize as initForA } from "./__generated_dbA__/fabbrica"; 
import { initialize as initForB } from "./__generated_dbB__/fabbrica";

initForA({ prisma: clientForA, multiple: true });
initForB({ prisma: clientForB, multiple: true });
```

How about this?

**Comment by yukukotani** [comment]
@Quramy Yes, we have multiple `schema.prisma` files.

In my opinion it might be better to separate instances for each client object by default, but since it can be a breaking change, `multiple` option seems nice!

**Comment by Quramy** [comment]
Thanks for your reply.

> In my opinion it might be better to separate instances for each client object by default, but since it can be a breaking change in some edge cases

As you said, almost all users will not be affected by prisma client instance separation.  I'll consider to "separate by default" .

**Comment by Quramy** [comment]
I released this feature as v2.1.0 .

If you want detail, see #257 description.
